etcd/etcd-env.sh

```diff
@@ -6,11 +6,11 @@ if test x$NAME == x; then
 fi
 export ETCD_NAME=${ETCD_NAME:-$HOSTNAME}
 export ETCD_DATA_DIR=/var/lib/etcd/${NAME}.etcd
```

Comment:
This is how I got it to work -- there's a relationship between what's going on in config.json.template, etcd-env.sh, and tmpfiles.template where the data dir's name is tied to the name we give to the container when we install it. I've tried a couple of variations without success. Can you shed additional light on this, @giuseppe